src/main/java/org/isf/operation/model/OperationRow.java

```diff
@@ -56,27 +56,27 @@ public class OperationRow {
     @JoinColumn(name = "OPER_OPE_ID_A")
     private Operation operation;
 
-    @NotNull
+    //@NotNull
     @Column(name = "OPER_PRESCRIBER")
     private String prescriber;
 
-    @NotNull
+    //@NotNull
     @Column(name = "OPER_RESULT")
     private String opResult;
 
-    @NotNull
+    //@NotNull
     @Column(name = "OPER_OPDATE")       // SQL type: datetime
     private LocalDateTime opDate;
 
     @Column(name = "OPER_REMARKS")
     private String remarks;
 
-    @NotNull
+    //@NotNull
     @ManyToOne
     @JoinColumn(name = "OPER_ADMISSION_ID")
     private Admission admission;
 
-    @NotNull
+    //@NotNull
     @ManyToOne
```

Comment:
@ArnaudFonzam why we commented all @NotNull annotations?

We should be consistent with DDL:

```
CREATE TABLE OPERATIONROW(
	OPER_ID_A INT (11) NOT NULL AUTO_INCREMENT,
	OPER_ID VARCHAR (11) NOT NULL,
	OPER_PRESCRIBER VARCHAR (150) NOT NULL,
	OPER_RESULT VARCHAR (250) NOT NULL,
	OPER_OPDATE DATETIME NOT NULL,
	OPER_REMARKS VARCHAR (250) NOT NULL,
	OPER_ADMISSION_ID INT(11) DEFAULT NULL, 
	OPER_OPD_ID INT(11) DEFAULT NULL,
	OPER_BILL_ID INT(11) DEFAULT NULL,
	OPER_TRANS_UNIT FLOAT NULL DEFAULT 0,	
	PRIMARY KEY (OPER_ID_A)
);
```

Reply:
because in the front-en part there properties are not mandatory, like admission or opd when you create operationRow we fix one of those two property for the rest you can fix them mandatories.